bin/pipeline_result_processor/prp/cli.py

```diff
@@ -0,0 +1,81 @@
+import click
+import logging
+from logging.config import dictConfig
+import pandas as pd
+import json
+from .parse import parse_resistance_pred, parse_virulence_pred, parse_qust_results, parse_cgmlst_results, parse_mlst_results
+
+dictConfig(
+    {
+        "version": 1,
+        "disable_existing_loggers": False,
+        "formatters": {
+            "default": {
+                "format": "[%(asctime)s] %(levelname)s in %(module)s: %(message)s",
+            }
+        },
+        "root": {"level": "INFO"},
+    }
+)
+LOG = logging.getLogger(__name__)
```

Comment:
Consider starting log at init level to be able to propagate log over all of package